CHANGELOG.md

```diff
@@ -1,3 +1,28 @@
+v2.14.0
+=======
+* added `OnTypeMismatch.IGNORE` which ignores specification containing mismatched parameter (except `spec = In.class` - in this specification only mismatched parameter values are ignored).
```

Comment:
in this specification only mismatched parameter values are ignored, but other ones which are valid are used to build a Specification).